pkg/kernel/kernel.go

```diff
@@ -38,6 +38,19 @@ func (k Params) Get(key string) ([]string, bool) {
 	return v, ok
 }
 
+func (k Params) GetOne(key string) (string, bool) {
+	all, found := k.Get(key)
+	if !found {
+		return "", false
+	}
+
+	if len(all) == 0 {
+		return "", false
+	}
+
+	return all[len(all)-1], true
```

Comment:
does this mean it returns the last item in case of the item passed multiple times?

Reply:
yes, if the same param is provided multiple times via a cmdline it's overridden by the last value provided. This way u can override hardcoded params in the image with whatever values u provide during booting